### PR TITLE
Add constants

### PR DESCRIPTION
**Pull request by operasfantom**
Closing #14

### CODE REVIEW

File: rd-kt/rd-framework/src/jvmTest/kotlin/com/jetbrains/rd/framework/test/cases/demo/model/DemoModel.Generated.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ import kotlin.reflect.KClass
 
 class DemoModel private constructor(
     private val _boolean_property: RdOptionalProperty<Boolean>,
```
Comment (korifey) [inline rd-kt/rd-framework/src/jvmTest/kotlin/com/jetbrains/rd/framework/test/cases/demo/model/DemoModel.Generated.kt]: remove generated model

File: rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt [hunk 2]
```diff
@@ -513,7 +533,19 @@ open class Kotlin11Generator(
                     member.isNullable -> "null"
                     else -> null
                 }
-//                is Member.Reactive.Stateful.Extension -> member.delegatedBy.sanitizedName(containing) + "()"
+                is Member.Const -> {
+                    val value = member.value
+                    when (member.type) {
+                        is PredefinedType.char -> """'$value'"""
+                        is PredefinedType.string -> """"$value""""
```
Comment (korifey) [inline rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt]: проверить экранирование

File: versions.gradle [hunk 3]
```diff
@@ -1,4 +1,4 @@
-ext.kotlin_version='1.3.11'
+ext.kotlin_version='1.3.31'
```
Comment (korifey) [inline versions.gradle]: 21

File: rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt [hunk 4]
```diff
@@ -398,34 +403,48 @@ open class Kotlin11Generator(
                 " */\n"
     }
```
Comment (korifey) [inline rd-kt/rd-gen/src/main/kotlin/com/jetbrains/rd/generator/nova/kotlin/Kotlin11Generator.kt]: const as default

File: rd-kt/rd-gen/models.gradle [hunk 5]
```diff
@@ -14,7 +14,7 @@ class GenerateTask extends JavaExec {
     GenerateTask() {
         super()
         group = 'generate'
-        main = 'com.jetbrains.rd.generator.nova.MainKt'
+        main = 'com.jetbbrains.rd.generator.nova.MainKt'
```
Comment (auduchinok) [inline rd-kt/rd-gen/models.gradle]: A typo?

Comment (operasfantom) [reply rd-kt/rd-gen/models.gradle]: Yeah. Fixed

**Comment by korifey** [comment]
Please merge it manually.

**Comment by operasfantom** [comment]
Merged.

**Pull request closed without merging**